src/setup/stateful.rs

```diff
@@ -0,0 +1,72 @@
+//! Contains code to start and stop a node with preloaded ledger data.
+
+use std::{
+    net::{Ipv4Addr, SocketAddr, SocketAddrV4},
+    path::PathBuf,
+    process::Child,
+};
+
+use crate::setup::{config::NodeMetaData, process, process::wait_for_start};
+
+const STATEFUL_PORT: u16 = 5005;
+
+/// Stateful node. Binds to 127.0.0.1:5005, loads data and configuration from `~/.ziggurat/ripple_stateful`.
+/// Does *not* clean its database after stopping.
+pub struct StatefulNode {
```

Comment:
why an extra object? I'd much rather just have a flag on the plain `Node`

Reply:
Removed